frontend/components/forms/validators/valid_password/index.ts

```diff
@@ -3,10 +3,23 @@ const NUMBER_PRESENT = /[0-9]+/;
 const SYMBOL_PRESENT = /\W+/;
 
 export default (password = "") => {
-  return (
-    password.length >= 12 &&
-    LETTER_PRESENT.test(password) &&
-    NUMBER_PRESENT.test(password) &&
-    SYMBOL_PRESENT.test(password)
-  );
+  let error = "";
+  let error_code = "";
+  if (password.length < 12) {
+    error = "Password must be at least 12 characters";
+    error_code = "too_short";
```

Comment:
`error_code` isn't used right now but exists in case the caller wants to use their own error message instead of the default supplied in `error`.